test/Microsoft.Extensions.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -246,7 +246,7 @@ public void GetDefaultBasePathForSources()
 #if DNXCORE50
             filePath = AppContext.BaseDirectory;
 #else
-            filePath = Path.GetFullPath((string)AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY"));
+            filePath = Path.GetFullPath((string)AppDomain.CurrentDomain.BaseDirectory);
```

Comment:
Right the comment was to update the src to remove references to the property:

* https://github.com/aspnet/Configuration/blob/73e162012a12b478502c4757b8416ad26ef3a0de/src/Microsoft.Extensions.Configuration.FileProviderExtensions/ConfigurationRootExtensions.cs#L25
* https://github.com/aspnet/Configuration/blob/73e162012a12b478502c4757b8416ad26ef3a0de/src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationExtensions.cs